2024-02-13 21:53:32.754 [INF] User profile is available. Using 'C:\Users\danix\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2024-02-13 21:53:32.980 [INF] Now listening on: http://localhost:5296
2024-02-13 21:53:32.983 [INF] Application started. Press Ctrl+C to shut down.
2024-02-13 21:53:32.984 [INF] Hosting environment: Development
2024-02-13 21:53:33.185 [INF] Executing endpoint '/Index'
2024-02-13 21:53:33.197 [INF] Route matched with {page = "/Index"}. Executing page /Index
2024-02-13 21:53:33.271 [INF] Executed page /Index in 71.2904ms
2024-02-13 21:53:33.273 [INF] Executed endpoint '/Index'
2024-02-13 21:53:33.343 [INF] Executing endpoint '/identifyClient/negotiate'
2024-02-13 21:53:33.352 [INF] Executed endpoint '/identifyClient/negotiate'
2024-02-13 21:53:33.355 [INF] Executing endpoint '/identifyClient'
2024-02-13 21:53:35.687 [INF] Executed endpoint '/identifyClient'
2024-02-13 21:53:37.159 [INF] Executing endpoint '/Index'
2024-02-13 21:53:37.160 [INF] Route matched with {page = "/Index"}. Executing page /Index
2024-02-13 21:53:37.163 [INF] Executed page /Index in 2.9965ms
2024-02-13 21:53:37.163 [INF] Executed endpoint '/Index'
2024-02-13 21:53:37.205 [INF] Executing endpoint '/identifyClient/negotiate'
2024-02-13 21:53:37.205 [INF] Executed endpoint '/identifyClient/negotiate'
2024-02-13 21:53:37.221 [INF] Executing endpoint '/identifyClient'
2024-02-13 21:53:43.406 [ERR] NoContentCliente não identificado. 
2024-02-13 21:53:43.413 [ERR] Failed to invoke hub method 'IdentifyMessage'.
System.AggregateException: One or more errors occurred. (ERR04: Cliente não identificado/inválido.)
 ---> System.ArgumentException: ERR04: Cliente não identificado/inválido.
   at HubIdentificacao.src.App.Controllers.IdentifyAPI.GetIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Controllers\IdentifyAPI.cs:line 79
   at HubIdentificacao.src.App.Services.ServiceIdentify.GetAPIIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Services\ServiceIdentify.cs:line 33
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at HubIdentificacao.src.App.Hubs.IdentifyHub.IdentifyMessage(String documento, String agencia, String dataHora) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Hubs\IdentifyHub.cs:line 59
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.ExecuteMethod(ObjectMethodExecutor methodExecutor, Hub hub, Object[] arguments)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.<Invoke>g__ExecuteInvocation|18_0(DefaultHubDispatcher`1 dispatcher, ObjectMethodExecutor methodExecutor, THub hub, Object[] arguments, AsyncServiceScope scope, IHubActivator`1 hubActivator, HubConnectionContext connection, HubMethodInvocationMessage hubMethodInvocationMessage, Boolean isStreamCall)
2024-02-13 21:53:44.297 [ERR] NoContentCliente não identificado. 
2024-02-13 21:53:44.298 [ERR] Failed to invoke hub method 'IdentifyMessage'.
System.AggregateException: One or more errors occurred. (ERR04: Cliente não identificado/inválido.)
 ---> System.ArgumentException: ERR04: Cliente não identificado/inválido.
   at HubIdentificacao.src.App.Controllers.IdentifyAPI.GetIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Controllers\IdentifyAPI.cs:line 79
   at HubIdentificacao.src.App.Services.ServiceIdentify.GetAPIIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Services\ServiceIdentify.cs:line 33
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at HubIdentificacao.src.App.Hubs.IdentifyHub.IdentifyMessage(String documento, String agencia, String dataHora) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Hubs\IdentifyHub.cs:line 59
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.ExecuteMethod(ObjectMethodExecutor methodExecutor, Hub hub, Object[] arguments)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.<Invoke>g__ExecuteInvocation|18_0(DefaultHubDispatcher`1 dispatcher, ObjectMethodExecutor methodExecutor, THub hub, Object[] arguments, AsyncServiceScope scope, IHubActivator`1 hubActivator, HubConnectionContext connection, HubMethodInvocationMessage hubMethodInvocationMessage, Boolean isStreamCall)
2024-02-13 21:53:44.992 [ERR] NoContentCliente não identificado. 
2024-02-13 21:53:44.993 [ERR] Failed to invoke hub method 'IdentifyMessage'.
System.AggregateException: One or more errors occurred. (ERR04: Cliente não identificado/inválido.)
 ---> System.ArgumentException: ERR04: Cliente não identificado/inválido.
   at HubIdentificacao.src.App.Controllers.IdentifyAPI.GetIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Controllers\IdentifyAPI.cs:line 79
   at HubIdentificacao.src.App.Services.ServiceIdentify.GetAPIIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Services\ServiceIdentify.cs:line 33
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at HubIdentificacao.src.App.Hubs.IdentifyHub.IdentifyMessage(String documento, String agencia, String dataHora) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Hubs\IdentifyHub.cs:line 59
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.ExecuteMethod(ObjectMethodExecutor methodExecutor, Hub hub, Object[] arguments)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.<Invoke>g__ExecuteInvocation|18_0(DefaultHubDispatcher`1 dispatcher, ObjectMethodExecutor methodExecutor, THub hub, Object[] arguments, AsyncServiceScope scope, IHubActivator`1 hubActivator, HubConnectionContext connection, HubMethodInvocationMessage hubMethodInvocationMessage, Boolean isStreamCall)
2024-02-13 21:53:45.620 [ERR] NoContentCliente não identificado. 
2024-02-13 21:53:45.621 [ERR] Failed to invoke hub method 'IdentifyMessage'.
System.AggregateException: One or more errors occurred. (ERR04: Cliente não identificado/inválido.)
 ---> System.ArgumentException: ERR04: Cliente não identificado/inválido.
   at HubIdentificacao.src.App.Controllers.IdentifyAPI.GetIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Controllers\IdentifyAPI.cs:line 79
   at HubIdentificacao.src.App.Services.ServiceIdentify.GetAPIIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Services\ServiceIdentify.cs:line 33
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at HubIdentificacao.src.App.Hubs.IdentifyHub.IdentifyMessage(String documento, String agencia, String dataHora) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Hubs\IdentifyHub.cs:line 59
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.ExecuteMethod(ObjectMethodExecutor methodExecutor, Hub hub, Object[] arguments)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.<Invoke>g__ExecuteInvocation|18_0(DefaultHubDispatcher`1 dispatcher, ObjectMethodExecutor methodExecutor, THub hub, Object[] arguments, AsyncServiceScope scope, IHubActivator`1 hubActivator, HubConnectionContext connection, HubMethodInvocationMessage hubMethodInvocationMessage, Boolean isStreamCall)
2024-02-13 21:53:46.313 [ERR] NoContentCliente não identificado. 
2024-02-13 21:53:46.314 [ERR] Failed to invoke hub method 'IdentifyMessage'.
System.AggregateException: One or more errors occurred. (ERR04: Cliente não identificado/inválido.)
 ---> System.ArgumentException: ERR04: Cliente não identificado/inválido.
   at HubIdentificacao.src.App.Controllers.IdentifyAPI.GetIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Controllers\IdentifyAPI.cs:line 79
   at HubIdentificacao.src.App.Services.ServiceIdentify.GetAPIIdentifyClient(Data dados) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Services\ServiceIdentify.cs:line 33
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at HubIdentificacao.src.App.Hubs.IdentifyHub.IdentifyMessage(String documento, String agencia, String dataHora) in C:\Users\danix\OneDrive\Documentos\Projetos\HubIdentify\HubIdentificacao\src\App\Hubs\IdentifyHub.cs:line 59
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.ExecuteMethod(ObjectMethodExecutor methodExecutor, Hub hub, Object[] arguments)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.<Invoke>g__ExecuteInvocation|18_0(DefaultHubDispatcher`1 dispatcher, ObjectMethodExecutor methodExecutor, THub hub, Object[] arguments, AsyncServiceScope scope, IHubActivator`1 hubActivator, HubConnectionContext connection, HubMethodInvocationMessage hubMethodInvocationMessage, Boolean isStreamCall)
2024-02-13 21:55:06.958 [INF] Application is shutting down...
2024-02-13 21:55:06.958 [INF] Executed endpoint '/identifyClient'
2024-02-13 21:55:06.961 [INF] Executing endpoint '/identifyClient/negotiate'
2024-02-13 21:55:06.961 [INF] Executed endpoint '/identifyClient/negotiate'
